doc/snippets/Microsoft.Data.SqlClient/SqlBulkCopy.xml

```diff
@@ -471,6 +471,25 @@ and faster to use a Transact-SQL `INSERT … SELECT` statement to copy the data.
 ]]></format>
 			</remarks>
 		</SqlRowsCopied>
+		<RowsCopied>
+			<summary>
+				The number of rows processed in the ongoing bulk copy operation.
+			</summary>
+			<value>
+				The integer value of the
+				<see cref="P:Microsoft.Data.SqlClient.SqlBulkCopy.RowsCopied" />
+				property.
+			</value>
+			<remarks>
+				<format type="text/markdown"><![CDATA[
+
+## Remarks
+This value is incremented during the <xref:Microsoft.Data.SqlClient.SqlBulkCopy.SqlRowsCopied> event and does not imply that these many rows have been sent to the server or committed.
```

Comment:
I would rephrase this slightly:
... and does not imply that this number of rows has been sent to the server or committed.